packages/subcommands/package.json

```
@@ -0,0 +1,78 @@
+{
+	"name": "@sapphire/plugin-subcommands",
+	"description": "Plugin for @sapphire/framework that makes it possible to create subcommands",
+	"version": "1.0.0",
```

```suggestion
	"version": "0.0.1",
```

Version will be set by Lerna